frontend/benefit/applicant/src/pages/login.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, IconSignin, Notification } from 'hds-react';
+import noop from 'lodash/noop';
+import { GetStaticProps, NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import Container from 'shared/components/container/Container';
+import Layout from 'shared/components/Layout';
```

Comment:
Yes, it is from Kesäseteli. As for the design, the decision was to _borrow_  yours for the time being since we don't have a proper one yet. But this may change in the future; that's why I kept the components separate.